docs/source/plotting/MeshPlotHelp.rst

```diff
@@ -20,136 +18,38 @@
 
 These sample shapes can be created with :ref:`algm-SetSample`, :ref:`algm-LoadSampleShape` or
 :ref:`algm-LoadSampleEnvironment` and copied using :ref:`algm-CopySample`.
-
-
-MeshObject
-##########
-
-Plot a MeshObject from an .stl (or .3mf) file:
-
-.. plot::
-   :include-source:
-
-    from mantid.simpleapi import *
-    import matplotlib.pyplot as plt
-    import numpy as np
-
-    from mpl_toolkits.mplot3d.art3d import Poly3DCollection
-    from mantid.api import AnalysisDataService as ADS
-
-    # load sample shape mesh file for a workspace
-    ws = CreateSampleWorkspace()
-    # alternatively: ws = Load('filepath') or ws = ADS.retrieve('ws')
-    ws = LoadSampleShape(ws, "tube.stl")
-
-    # get shape and mesh vertices
-    sample = ws.sample()
-    shape = sample.getShape()
-    mesh = shape.getMesh()
-
-    # Create 3D Polygon and set facecolor
-    mesh_polygon = Poly3DCollection(mesh, facecolors = ['g'], edgecolors = ['b'], alpha = 0.5, linewidths=0.1)
-
-    fig, axes = plt.subplots(subplot_kw={'projection':'mantid3d'})
-    axes.add_collection3d(mesh_polygon)
-
-    axes.set_mesh_axes_equal(mesh)
-    axes.set_title('Sample Shape: Tube')
-    axes.set_xlabel('X / m')
-    axes.set_ylabel('Y / m')
-    axes.set_zlabel('Z / m')
-
-    plt.show()
-
-
-CSGObject
-#########
-
 For help defining CSG Shapes and Rotations, see :ref:`HowToDefineGeometricShape`.
-Plot a CSGObject defined in an XML string:
-
-.. plot::
-   :include-source:
-
-   # import mantid algorithms, numpy and matplotlib
-   from mantid.simpleapi import *
-   import matplotlib.pyplot as plt
-   import numpy as np
-   from mpl_toolkits.mplot3d.art3d import Poly3DCollection
-
-   ws = CreateSampleWorkspace()
-
-   merge_xml = ' \
-   <cylinder id="stick"> \
-   <centre-of-bottom-base x="-0.5" y="0.0" z="0.0" /> \
-   <axis x="1.0" y="0.0" z="0.0" />  \
-   <radius val="0.05" /> \
-   <height val="1.0" /> \
-   </cylinder> \
-   \
-   <sphere id="some-sphere"> \
-   <centre x="0.7"  y="0.0" z="0.0" /> \
-   <radius val="0.2" /> \
-   </sphere> \
-   \
-   <rotate-all x="90" y="-45" z="0" /> \
-   <algebra val="some-sphere (: stick)" /> \
-   '
-
-   SetSample(ws, Geometry={'Shape': 'CSG', 'Value': merge_xml})
-
-   sample = ws.sample()
-   shape = sample.getShape()
-   mesh = shape.getMesh()
 
-   mesh_polygon = Poly3DCollection(mesh, edgecolors = 'blue', linewidths=0.1)
-   mesh_polygon.set_facecolor((1,0,0,0.5))
+Workbench
+#########
 
-   fig, axes = plt.subplots(subplot_kw={'projection':'mantid3d'})
-   axes.add_collection3d(mesh_polygon)
+To quickly plot all Sample, Container and Component shapes attached to a workspace, right-click on it
+in the Workspaces Toolbox and select ``Show Sample Shape``. A black arrow will be added for the beam direction
+and, if there is a UB matrix, colored arrows for the real (solid line) and reciprocal (dashed line) lattice vectors.
```

Comment:
Just wondering if it's meaningful to define which colours are used for which lattice vectors? So the first reciprocal lattice vector is the one associated with the h. So something like red=h, green=k, blue=l. But not sure if there's other better terminology here. Perhaps the first reciprocal lattice vector is commonly called a1 or something similar. Perhaps check with Richard

Reply:
I think you're right I should call the reciprocal b1, b2, b3 and real a1, a2, a3.